bangazonreports/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from .views import expensive_list
+from .views import expensive_list, Cheap_list, completed_order_list
 
 urlpatterns = [
     path('reports/expensive', expensive_list),
-]
+    path('reports/cheap', Cheap_list),
```

Comment:
`Cheap_list` cannot be imported. Not finding it anywhere but this file.

Reply:
I'd accidentally branched this off another PR and then tried to remove it and didn't get everything removed. Should be fixed now.